Gruntfile.js

```diff
@@ -81,7 +84,7 @@ module.exports = (grunt) => {
       options: { fix: true },
     },
     qunit: {
-      files: ['tests/flow.html'],
+      files: ['tests/flow.html', 'tests/flow_ts.html'],
```

Comment:
I personally don't see a benefit of having 2 flow html files

Reply:
It is very simple and does not have drawbacks. One benefit might be that it provides a _**nice visual feedback on the progress**_ of the migration and in the end when everything is migrated we will get one file anyway. 
But now you managed one file. Let us review it!

Reply:
A drawback is a code repetition, because we have 2 identical files.

Reply:
After reviewing #873, my oppinion is this approach(#872) has the following:

**benefits**:
- it is **very simple**, simpler than the one file approach
- it provides **visual feedback on the progress of the migration**

**drawbacks**:
- code repetition on the **two html files**, but one will no longer be needed at the end of the migration.

Personally I prefer this approach but I also made it :)

@0xfe @mscuthbert you decide, in the end of the day it is a matter of taste and both approaches do work.

Reply:
it is not only `flow_ts.html`, which makes a confusion, you also have `vexflow-test-ts.js` and `run_ts.ts` - these are just ugly

Reply:
Thanks Rodrigo and Tim. Let's go with the single file please :-)

Reply:
@0xfe this is fine, I will leave this open until the other approach is merged should there be a show stopper. 

@0xfe I still do not understand the relation between `vexflow-tests.js` `vexflow-tests-node.js` **please review `Gruntfile.js`**, I am definetively confused :).

Reply:
@wassertim after the review #873 I believe that there are no longer show stoppers, I close this one.